app/_includes/snippets/gateway-operator/install_with_helm.md

```diff
@@ -12,10 +12,30 @@ helm repo update kong
 
 Install {{ site.kgo_product_name }} with Helm:
 
+{% if include.mode == "konnect" %}
+
+```bash
+helm upgrade --install kgo kong/gateway-operator -n kong-system --create-namespace --set image.tag={{ kgo_version }} --set env.ENABLE_CONTROLLER_KONNECT="true" --set kubernetes-configuration-crds.enabled="true"
```

Comment:
Nit for consideration: we could break these `helm` invocation lines into separate like so 

```suggestion
helm upgrade --install kgo kong/gateway-operator -n kong-system --create-namespace --set image.tag={{ kgo_version }} \
  --set env.ENABLE_CONTROLLER_KONNECT="true" \
  --set kubernetes-configuration-crds.enabled="true"
```

Making it easier to follow and understand which flags are set.

Reply:
+1 on this. Would really help readability.

Reply:
sg 👍